.eslintignore

```diff
@@ -1 +1,2 @@
+.quasar
```

Comment:
That missing line was triggering a warning in the admin-ui build stage of the tests pipeline.